package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "c",
+  "version": "v0.06",
+  "description": "Compile and execute C \'scripts\' in one go!",
+  "global": "true",
+  "install": "install c ${PREFIX:-/usr/local}/bin",
```

Comment:
It evaluates to the environment variable PREFIX if it is defined otherwise
it fallbacks to /usr/locals

On Wed, Mar 4, 2015, 7:30 PM Ryan Jacobs <notifications@github.com> wrote:

> In package.json
> <https://github.com/ryanmjacobs/c/pull/4#discussion_r25829490>:
>
> > @@ -0,0 +1,9 @@
> > +{
> > +  "name": "c",
> > +  "version": "v0.06",
> > +  "description": "Compile and execute C \'scripts\' in one go!",
> > +  "global": "true",
> > +  "install": "install c ${PREFIX:-/usr/local}/bin",
>
> Sorry if I am being naive, but what does ${PREFIX:-/usr/local}/bin do?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ryanmjacobs/c/pull/4/files#r25829490>.
>